.github/workflows/build-lint-push-containers.yml

```diff
@@ -30,26 +44,57 @@ jobs:
     runs-on: ubuntu-latest
     env:
       POETRY_VIRTUALENVS_CREATE: "false"
+
     steps:
       - name: Checkout
         uses: actions/checkout@v4
 
-      - name: Setup python (release)
-        if: github.event_name == 'release'
+      - name: Setup Python
         uses: actions/setup-python@v5
         with:
           python-version: ${{ env.PYTHON_VERSION }}
 
-      - name: Install dependencies (release)
-        if: github.event_name == 'release'
+      - name: Install Poetry
         run: |
           pipx install poetry
           pipx inject poetry poetry-bumpversion
 
+      - name: Get Prowler version
+        run: |
+          PROWLER_VERSION="$(poetry version -s 2>/dev/null)"
+
+          # Store prowler version major just for the release
+          PROWLER_VERSION_MAJOR="${PROWLER_VERSION%%.*}"
+          echo "PROWLER_VERSION_MAJOR=${PROWLER_VERSION_MAJOR}" >> "${GITHUB_ENV}"
+
+          case ${PROWLER_VERSION_MAJOR} in
```

Comment:
This is returning the same in both case statements

Reply:
Or it is just a way to set the major for just the v3 and v4?

Reply:
It's just prepared for the v4 release, as you can see the commented code will be uncommented for the v4 release:
```
3)
              # TODO: update it for v3 and v4
              # echo "LATEST=v3-latest" >> "${GITHUB_ENV}"
              # echo "STABLE_TAG=v3-stable" >> "${GITHUB_ENV}"
              echo "LATEST=latest" >> "${GITHUB_ENV}"
              echo "STABLE_TAG=stable" >> "${GITHUB_ENV}"
              ;;

          # TODO: uncomment for v3 and v4
          # 4)
          #     echo "LATEST=latest" >> "${GITHUB_ENV}"
          #     echo "STABLE_TAG=stable" >> "${GITHUB_ENV}"
          #     ;;
```